tests/run-pass/panic-default-flags.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    panic!("Miri panic!");
+}
```

Comment:
Just `panic.rs` seems like a reasonable test name. Or maybe `panic1.rs`; could you add a `panic2` that uses string formatting? IIRC these have different code paths (that's why we have 4 compile-fail panic tests).